backend/methods/relative_superiority.py

```diff
@@ -33,9 +33,13 @@ def relative_superiority(m_votes, v_desired_row_sums, v_desired_col_sums,
             first_in.append(next_in)
 
             # Calculate continuation:
-            _, div_after = apportion1d(m_votes[c],
+            new_votes = copy(m_votes[c])
+            new_votes[next_in] = alloc_next[2]
```

Comment:
This was removed here:
https://github.com/PeturOA/voting/commit/969501eac131f34fad6aa010857d84cd2e8af9cf#diff-023f21b6e38229a5195b51cfb4040e82

(more specifically, here:
https://github.com/PeturOA/voting/commit/969501eac131f34fad6aa010857d84cd2e8af9cf#diff-023f21b6e38229a5195b51cfb4040e82R32
)

But removing it didn't really fix the bug.